apps/examples/wifi_manager/wm_test/wm_test_stress_test5.c

```diff
@@ -0,0 +1,62 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <wifi_manager/wifi_manager.h>
+#include "wm_test.h"
+
+#define MAX_INTEROP_AP_CONFIG_COUNT 10
+#define MAX_INTEROP_SECURITY_LEN 10
+
+typedef struct {
+	wifi_manager_ap_config_s ap_config;
+	char security[MAX_INTEROP_SECURITY_LEN];
+	int scan_result;
+	int connect_result;
+} interop_ap_info_s;
+
+typedef struct {
+	int ap_count;
+	interop_ap_info_s ap_info[MAX_INTEROP_AP_CONFIG_COUNT];
+} interop_ap_config_list_s;
+
+static interop_ap_config_list_s g_apconfig_list;
+
+void wm_run_stress_test5(struct wt_options *opt)
```

Comment:
how about to modify file name and function name. because stress_test5 calls stress_test2 which is same level. so it seems not appropriate. I think stress_test5 is above stress_test2.
After all entry of wm_run_stress_test2 need to be 1 not multiple(stress5 and wm_run_stress_test)
I think you can achieve this by modifying _wt_parse_stress can handle both stress2 and stress5 parameters.

Reply:
Sir, I have updated my commit as suggested. Please review it.